Kickstarter-iOS/DataSources/CommentRepliesDataSourceTests.swift

```diff
@@ -0,0 +1,27 @@
+@testable import Kickstarter_Framework
+@testable import KsApi
+@testable import Library
+import Prelude
+import XCTest
+
+class CommentRepliesDataSourceTests: XCTestCase {
+  let dataSource = CommentRepliesDataSource()
+  let tableView = UITableView()
+  let commentSection = CommentRepliesDataSource.Section.comment.rawValue
+
+  override func setUp() {
+    super.setUp()
+    self.dataSource.load(comment: .template, project: .template)
+  }
+
+  func testLoadedComments() {
```

Comment:
We probably want these tests to be a bit more robust than checking a count of one for the number of items and sections.

Reply:
Ok, I added a test for the model data that comes back from the data source.